_rules/first-focusable-element-is-link-to-non-repeated-content-8a213c.md

```diff
@@ -0,0 +1,658 @@
+---
+id: 8a213c
+name: First focusable element is link to non-repeated content
+rule_type: atomic
+description: |
+  This rule checks that the first focusable element is a link to non-repeated content in the page
+accessibility_requirements:
+  wcag-technique:G1: # Adding a link at the top of each page that goes directly to the main content area
+    forConformance: false
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility Tree
+  - CSS styling
+  - DOM Tree
+  - Language
+acknowledgments:
+  authors:
+    - Christina Adams
+    - Jean-Yves Moyen
+  assets:
+    - _The Three Kingdoms_ by Luo Guanzhong, translation by Yu Sumei (Tuttle publishing, 2014, ISBN 9780804843935)
+---
+
+## Applicability
+
+This rule applies to any [HTML web page][].
+
+## Expectation 1
+
+Each test target has a non-empty [sequential focus navigation order][sequential focus navigation].
```

Comment:
Don't think we need this expectation. There is no scenario in which expectation 1 is false, but expectation 2 is true.